app/main.py

```diff
@@ -1,2 +1,153 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 4
+        self.overflow_limit = self.capacity * 2 // 3
+        self.len = 0
+        self.container = [None] * self.capacity
+
+    def resize(self, size: int) -> None:
+        container = self.container
+        self.capacity = size
+        self.overflow_limit = self.capacity * 2 // 3
```

Comment:
* 2 // 3 is some magic number. Create some function and use this magic number only in this function